.github/workflows/ci.yml

```diff
@@ -94,3 +94,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: dtolnay/rust-toolchain@clippy
       - run: cargo clippy --workspace --tests -- -Dclippy::all
+
+    clang-tidy:
```

Comment:
This yaml block is misaligned so it hasn't run in CI.

![Screenshot from 2020-11-23 17-09-05](https://user-images.githubusercontent.com/1940490/100033231-a4f4d400-2dae-11eb-8cfd-0a76b93d90b4.png)